cluster/gce/gci/configure.sh

```diff
@@ -546,6 +547,42 @@ function install-containerd-ubuntu {
   sudo systemctl start containerd
 }
 
+function install-auth-provider-gcp {
+  local -r auth_provider_tar="auth-provider-gcp-${HOST_PLATFORM}_amd64.tar.gz"
+  echo "Downloading auth-provider-gcp ${auth_provider_tar}" .
+
+  local -r auth_provider_release_path="https://storage.googleapis.com/cloud-provider-gcp"
+  download-or-bust "${DEFAULT_AUTH_PROVIDER_GCP_HASH_AMD64}" "${auth_provider_release_path}/${auth_provider_tar}"
+
+  # Keep in sync with --image-credential-provider-bin-dir in ../util.sh
+  local auth_provider_dir="${KUBE_HOME}/auth-provider-gcp"
+  mkdir -p "${auth_provider_dir}"
+  tar xzf "${KUBE_HOME}/${auth_provider_tar}" -C "${auth_provider_dir}" --overwrite
+  mv "${auth_provider_dir}/auth-provider-gcp" "${KUBE_BIN}"
```

Comment:
Do we need to `chmod +x` the binary e.g. like during npd installation? https://github.com/kubernetes/kubernetes/blob/5d446b2/cluster/gce/gci/configure.sh#L304